src/test/util_tests.cpp

```diff
@@ -78,6 +78,29 @@ BOOST_AUTO_TEST_CASE(util_datadir)
     BOOST_CHECK_EQUAL(dd_norm, args.GetDataDirBase());
 }
 
+namespace {
+class NoCopyOrMove
+{
+public:
+    int i = 0;
```

Comment:
```suggestion
    int i;
```

nit: Can leave un-initialized for clarity, since the default value is never used.